docs/styles.css

```diff
@@ -0,0 +1,216 @@
+* {
+    box-sizing: border-box;
+    padding:0;
+    margin:0;
+    
+    
+    
+}
+
+body{
+    height: 100vh;
+    background-image: url(assets/bg-desktop.png);
+    background-size: 100vw 50vh ;
+    background-repeat: no-repeat;
+    background-position: bottom;
+    background-color:hsl(229, 57%, 11%); 
+    display:flex;
+    justify-content: center;
+    flex-direction: column;
+    font-family: 'Raleway', sans-serif;
+    font-size: 1rem;
+    
+    
+    
+} 
+   
+.floatContainer{ 
+    height: 170px;
+    display:flex;
+    flex-direction:row;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+
+.fyloCard {
+    height:100%;
+    width:300px;
+    display: flex;
+    flex-direction: column;
+    background-color:hsl(228, 56%, 26%);
+    border-radius: 10px 90px 10px 10px;
+    margin-right: 1%;
+    
+}
+.fylohead{
+    margin-top:9%;
+    margin-left:10%;
+    height:50%;
+}
+.fyloimages{
+    height:50%;
+    margin-left:10%;
+    display: flex;
+    justify-content: space-between;
+    width:50%;
+    
+}
+.iconBox{
+    display:flex;
+    justify-content: center;
+    align-items:center;
+    background-color: hsl(228, 59%, 15%);
+    height: 40px;
+    width: 40px;
+    border-radius:7px;
+    
+}
+
+
+.icons{
+    height:17px;
+        
+}
+.icon-1{
+    height:20px;
+}
+
+
+.storage{
+    height:100%;
+    width: 550px;
+    
+}
+.storageInt{
+    
+    margin-top: 7%;
+    height:77.5%;
+    background-color:hsl(228, 56%, 26%);    
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.dialogue{
+    width:100%;
+    display: flex;
+    
+}
+
+.dialoguebox{
+    width:490px;
+    height:10%;
+    position:absolute;
```

Comment:
can you show me what the relative container for this absolute class name?

Reply:
Originally I thought that it was always position based on the div it was inside (so div class=dialogue) but after reading up I see that it is based on the nearest positioned ancestor and due to how I formatted the page it seems to work anyways. Is there a better way to get elements to lay on top of each other to create the popup with the 185GB left? Or is the div class=dialogue positioned based on flex and so it is working as intended?